src/main/java/com/auth0/json/mgmt/jobs/Job.java

```diff
@@ -0,0 +1,56 @@
+package com.auth0.json.mgmt.jobs;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Class that represents an Auth0 Job object. Related to the {@link com.auth0.client.mgmt.JobsEntity} entity.
+ */
+@SuppressWarnings({"unused", "WeakerAccess"})
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Job {
+
+    @JsonProperty("status")
+    private String status;
+    @JsonProperty("type")
+    private String type;
+    @JsonProperty("created_at")
+    private String createdAt;
+    @JsonProperty("id")
+    private String id;
+
+    @JsonCreator
+    public Job(@JsonProperty("status") String status, @JsonProperty("type") String type, @JsonProperty("id") String id) {
```

Comment:
I think for the time being, this doesn't need to be public. Can you try making it private or package protected at least? See if Jackson complains.